src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -24,19 +24,19 @@
 public class AddTaskCommand extends Command {
 
     /* Message printed if wrong usage */
-    public static final String MESSAGE_USAGE = "addt" + ": Adds a task to the NUS Classes. "
+    public static final String MESSAGE_USAGE = "addt" + ": Adds a task to NUS Classes. "
             + "Parameters: "
             + PREFIX_TASKNAME + "TASKNAME "
             + PREFIX_DATETIME + "DATETIME [, ENDDATETIME]"
-            + PREFIX_TAG + "TAG "
-            + PREFIX_LINK + "LINK "
-            + PREFIX_RECURRING + "PERIOD RECURRENCE\n"
+            + " [" + PREFIX_TAG + "TAG] "
+            + " [" + PREFIX_LINK + "LINK] "
+            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE]\n"
             + "Example: " + "addt" + " "
             + PREFIX_TASKNAME + "John Doe "
```

Comment:
Probably can change the Task name example to "Conduct Lecture" instead of "John Doe" to prevent confusion?

Reply:
👌